### PR TITLE
Feature/1 simpacc UI

### PR DESCRIPTION
**Pull request by chemstone143**
- [ ] Refactor elements (Buttons, InputText, and the likes) into a separate and reusable react components.
- [ ] Make a sidebar menu as required by Simpacc team
- [ ] Test SyncFusion Grid.[ Click this to see example grid](https://ej2.syncfusion.com/react/demos/#/tailwind/grid/overview)
- [ ] Make the design responsive for small, medium, large design. [See this](https://github.com/dnycstr/simpacc-accounting-ui/pull/11#issuecomment-1680383124)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Go to the /build folder and run
 To run with your desired port
 
 > npx serve -p [PORT_NUMBER]
-
-Junex
-bleh!
+> jacko
+> yowwwn
+> Changes
+yada yada ydaa
```
Comment (dnycstr) [inline README.md:25]: Clean this up guys

File: src/modules/billing/pages/Revenue and Collections/Products&Services.tsx
Comment (dnycstr) [inline src/modules/billing/pages/Revenue and Collections/Products&Services.tsx:1]: Avoid using [SPACE] and Special Characters such as '&' for foldernames and filenames.

File: src/modules/billing/pages/Reports/Profit&Loss/Comparative.tsx
Comment (dnycstr) [inline src/modules/billing/pages/Reports/Profit&Loss/Comparative.tsx:1]: Avoid & in Foldername.

Rename the folder to "ProfitAndLoss"

File: src/modules/billing/pages/PurchaseandPayments/Vendors.tsx
Comment (dnycstr) [inline src/modules/billing/pages/PurchaseandPayments/Vendors.tsx:1]: Rename PurchaseandPayments to "PurchaseAndPayments"

File: src/modules/billing/pages/Reports/Profit&Loss/YeartoDate.tsx
Comment (dnycstr) [inline src/modules/billing/pages/Reports/Profit&Loss/YeartoDate.tsx:1]: Rename this to "YearToDate.tsx"

File: src/modules/billing/AppRouter.tsx [hunk 2]
```diff
@@ -1,12 +1,103 @@
...
+      {
+        path: 'purchase-&-payments/bills',
+        element: <Bills />,
+      },
+      {
+        path: 'purchase-&-payments/payment-set-up',
+        element: <PaymentsSetup />,
+      },
+
+      { path: 'purchase-&-payments/vendors', element: <Vendor /> },
```
Comment (dnycstr) [inline src/modules/billing/AppRouter.tsx:66]: Do not use '&' for path. Replace this with "purchase-and-payments". & character has special meaning in URLs.

**Review by dnycstr — CHANGES_REQUESTED** [review]

**Comment by dnycstr** [comment]
The design looks good on Desktop now. Nice work 👍

**Comment by dnycstr** [comment]
Make the design responsive for smaller device.
image

**Review by Jheyem0212 — APPROVED** [review]

**Pull request merged**